app/i18n/locales/en-US.json

```diff
@@ -117,7 +117,7 @@
   "wallet.restore.dialog.wallet.name.input.label": "Wallet name",
   "wallet.send.form.amount.equalsAda": "equals {amount} ADA",
   "wallet.send.form.amount.hint": "Amount in Lovelaces",
-  "wallet.send.form.amount.label": "Amount in Lovelaces",
+  "wallet.send.form.amount.label": "Amount",
   "wallet.send.form.description.hint": "You can type a message here, if you want.",
```

Comment:
Let's use `"wallet.redeem.dialog.adaAmountLabel": "Ada amount"` here since we have it translated to Japanese. Not the same translation key, but same copy: "Ada amount". So we can create Japanese translation without translators.